### PR TITLE
BCLD-3716 ~ Fix Message.SendMessage() API - CPP

### PR DESCRIPTION
**Pull request by ruth0048**

### CODE REVIEW

**Comment by ruth0048** [comment]
Also covers BCLD-3717, 3718, 3719, 3721

File: include/braincloud/BrainCloudMessaging.h [hunk 1]
```diff
@@ -90,11 +91,10 @@ namespace BrainCloud
 		 * Service Operation - SendMessage
 		 *
 		 * @param toProfileIds
-		 * @param messageText
-		 * @param messageSubject
+		 * @param contentJson the message you are sending
 		 * @param callback The method to be invoked when the server response is received
 		 */
-		void sendMessage(const std::vector<std::string>& in_toProfileIds, const std::string& in_messageText, const std::string& in_messageSubject, IServerCallback* in_callback = NULL);
+		void sendMessage(const std::vector<std::string>& in_toProfileIds, std::string contentJson, IServerCallback* in_callback = NULL);
```
Comment (davidstl) [inline include/braincloud/BrainCloudMessaging.h]:
```suggestion
		void sendMessage(const std::vector<std::string>& in_toProfileIds, std::string in_contentJson, IServerCallback* in_callback = NULL);
```

File: src/BrainCloudMessaging.cpp [hunk 2]
```diff
@@ -74,14 +74,11 @@ namespace BrainCloud
 		m_client->sendRequest(sc);
     }
 
-	void BrainCloudMessaging::sendMessage(const std::vector<std::string>& in_toProfileIds, const std::string& in_messageText, const std::string& in_messageSubject, IServerCallback* in_callback)
+	void BrainCloudMessaging::sendMessage(const std::vector<std::string>& in_toProfileIds, std::string contentJson, IServerCallback* in_callback)
```
Comment (davidstl) [inline src/BrainCloudMessaging.cpp]:
```suggestion
	void BrainCloudMessaging::sendMessage(const std::vector<std::string>& in_toProfileIds, std::string in_contentJson, IServerCallback* in_callback)
```

File: include/braincloud/BrainCloudMessaging.h [hunk 3]
```diff
@@ -7,6 +7,7 @@
 
 #include <string>
 #include <vector>
+#include "json/json.h"
```
Comment (davidstl) [inline include/braincloud/BrainCloudMessaging.h]:
```suggestion
```
Include that in the .cpp

**Pull request merged**